src/vs/workbench/contrib/terminalContrib/accessibility/browser/terminalAccessibleBuffer.ts

```diff
@@ -87,6 +81,11 @@ export class AccessibleBufferWidget extends DisposableStore {
 				this._font = _xterm.getFont();
 			}
 		}));
+		this.add(this._xterm.raw.onWriteParsed(async () => {
+			if (this._accessibleBuffer.classList.contains('active')) {
```

Comment:
Pulling 'active' into a const enum would be a good idea to prevent accidental typos

Reply:
and 'hide'